catalog/app/containers/Admin/Buckets.js

```diff
@@ -871,9 +878,23 @@ function CRUD({ buckets }) {
     },
   ]
 
+  const editingBucket = React.useMemo(
+    () => (bucketName ? rows.find(({ name }) => name === bucketName) : null),
```

Comment:
i think it might make sense to redirect to the root page or render error msg inside the dialog if the bucket wasnt found instead of simply ignoring the query param

Reply:
I made redirect, using react-router-dom's `<Redirect />`